packages/strapi-bookshelf/lib/index.js

```diff
@@ -361,7 +361,7 @@ module.exports = function(strapi) {
                             type = definition.client === 'pg' ? 'double precision' : 'double';
                             break;
                           case 'decimal':
-                            type = 'decimal';
```

Comment:
I think we should look at making this configurable in the adminUI and the config files if possible. 

Using this I believe the accuracy is 10 total digits with 2 being after the decimal point so something like:
`99999999.99`